### PR TITLE
Set default tester image value

### PR DESCRIPTION
**Pull request by eshiroma**
/gcbrun

### CODE REVIEW

**Comment by eshiroma** [comment]
Replace functionality added to `wordpress` but not `wordpress-updatable` added in #140 that was accidentally removed in #142.

**Comment by eshiroma** [comment]
Replaces the functionality added in #140, but accidentally removed in #142.

Note that the Dockerfile envsubst handles Verification's invocation of `mpdev verify`, but the `APP_TEST_PARAMETERS` build arg handles this repo's CloudBuild verification.

**Comment by eshiroma** [comment]
Confirmed successful verification run after ingesting built images.

**Review by gibbleyg — APPROVED** [review]

File: app_v2.Makefile [hunk 1]
```diff
@@ -102,20 +96,22 @@ app/uninstall: .build/var/APP_DEPLOYER_IMAGE \
                .build/var/APP_PARAMETERS
 	$(call print_target)
 	kubectl delete 'application/$(call name_parameter)' \
-	    --namespace='$(call namespace_parameter)' \
+	    --namespace='$(APP_PARAMETERS)' \
```
[the remark this answers is not in the document]
Comment (eshiroma) [reply app_v2.Makefile]: Yes, good catch! Fixed.

**Pull request merged**